packages/dev/core/src/Animations/animationGroup.ts

```diff
@@ -11,6 +11,7 @@ import { EngineStore } from "../Engines/engineStore";
 import type { AbstractScene } from "../abstractScene";
 import { Tags } from "../Misc/tags";
 import type { AnimationGroupMask } from "./animationGroupMask";
+import "./animatable";

```

Comment:
i really wonder how we missed this import so far :-)

Reply:
Yea, I guess for other usages animatable was already coming in from something else.